job-scheduler/src/main/resources/scheduler/db/migration/V2018_06_06_1515__add-quartz-tables.sql

```diff
@@ -0,0 +1,173 @@
+-- In your Quartz properties file, you'll need to set
+-- org.quartz.jobStore.driverDelegateClass = org.quartz.impl.jdbcjobstore.PostgreSQLDelegate
+
+CREATE TABLE qrtz_job_details
```

Comment:
Does it mean we will keep our claimstore db even when move towards ccd on all envs?

Reply:
no this is going into the cmc cnp database, 
it means we will have a database still

but its on CNP
its updated to be postgres 10
and the database name is cmc